res/layout/files.xml

```diff
@@ -18,9 +18,11 @@
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
   -->
 <android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                                        xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/drawer_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:fitsSystemWindows="true"

```

Comment:
Forget it. This is https://developer.android.com/reference/android/view/View.html#setFitsSystemWindows(boolean) , that's different; ```fitSystemWindows``` Vs ```fitsSystemWindows```. Come on, Google...